plgrid/string_templates.py

```diff
@@ -40,3 +40,7 @@
 
 exit 0
 """  # skipcq: FLK-E501
+
+plgdata_list_url: str = """{{http_plgdata}}/list/{{hostname}}/net/people/{{plguserlogin}}/sh_output/{{job_id}}"""
```

Comment:
The path here needs to be adjustr to $SCRATCH
we shouldn't save simulations in $HOME

Reply:
Let's work on it a bit later